omrsigcompat/omrsig.cpp

```diff
@@ -197,8 +197,12 @@ omrsig_handler(int sig, void *siginfo, void *uc)
 sighandler_t
 omrsig_primary_signal(int signum, sighandler_t handler)
 {
-	struct sigaction act = {{0}};
-	struct sigaction oldact = {{0}};
+	struct sigaction act;
+	struct sigaction oldact;
+
+	memset(&act, 0, sizeof(struct sigaction));
```

Comment:
The zero initializer gives a warning on clang?

Reply:
Yes, it does.